printer_mods/baccula/revo_skirt_nozzle_holder/README.md

```diff
@@ -0,0 +1,24 @@
+# Revo Nozzle Holder v2.4 350mm
+Remix of [Andre's](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/Andre/Revo_Nozzle_Skirt_Trident) skirt-mounted nozzle holder to fit Voron 2.4 350mm printers, the BOM and installation steps are the same.
```

Comment:
please use a relative link: `../../Andre/Revo_Nozzle_Skirt_Trident` to avoid breaking links in the future (might need more or less `..` I didn't count)